src/js/components/Meta.js

```diff
@@ -321,6 +350,11 @@ class Meta extends React.PureComponent {
               )}
             </h4>
           )}
+          {this.state.isShabadPlayable && (
+            <div ref={this.audioPlayerIconRef} role='button' className="meta-hukamnama-right" onClick={this.setHukamnamaAudioPlayerVisibility}>
```

Comment:
@MySmilingTurban  can use button directly here ?

Reply:
@gurjit03 
It can be used but this was taken from line 298 : Meta.js (Today's Hukamnama).
With the button tag it doesn't align to center.